app/handlers/ErrorHandler.scala

```diff
@@ -17,23 +17,24 @@
 package handlers
 
 import play.api.i18n.{I18nSupport, MessagesApi}
-import play.api.mvc.Request
+import play.api.mvc.{Request, RequestHeader}
 import play.twirl.api.Html
 import uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler
 import views.html.{ErrorTemplate, PageNotFoundView}
 
 import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class ErrorHandler @Inject()(
                               val messagesApi: MessagesApi,
                               view: ErrorTemplate,
                               notFoundView: PageNotFoundView
-                            ) extends FrontendErrorHandler with I18nSupport {
+                             )(implicit val ec: ExecutionContext) extends FrontendErrorHandler with I18nSupport {
```

Comment:
Did we need this `ExecutionContext` in the end?  It doesn't seem to be used here.

Reply:
If it's removed, this error occurs
`Class 'ErrorHandler' must either be declared abstract or implement abstract member 'ec: ExecutionContext' in 'uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler'`

Reply:
fair enough, then!